packages/python-packages/doc-warden/warden/index_packages.py

```diff
@@ -223,6 +223,8 @@ def get_net_packages_info(config):
     pkg_locations, ignored_pkg_locations = get_net_packages(config)
 
     for pkg_file in (pkg_locations + ignored_pkg_locations):
+        pkg_version = parse_csproj(pkg_file)
+
         pkg_name = os.path.splitext(os.path.basename(pkg_file))[0]
         if(pkg_name not in config.package_indexing_exclusion_list):
             changelog = find_above_file('changelog.md', pkg_file, config.get_package_indexing_traversal_stops(), net_early_exit, os.path.normpath(config.target_directory))
```

Comment:
Here and in other places in this tool we should change to finding exact case sensitive file names so "CHANGELOG.md" and "README.md" because there are a number of places we have links to these files and most web requests are case sensitive so it would be good to make them consistent so we don't have prob for the various different casings like https://github.com/Azure/azure-sdk/blob/master/eng/scripts/Generate-Release-Tables.ps1#L95

Reply:
I instead added a section to the scan that tests the case os each file to make sure its upper case README.md and CHANGELOG.md